webrtc.html

```diff
@@ -8793,57 +8793,49 @@ <h3><dfn>RTCIceTransport</dfn> Interface</h3>
         </li>
```

Comment:
Now that we've merged the state and pair update as per #2283, we need to update the above text "When the ICE Agent indicates that the RTCIceTransportState for an RTCIceTransport has changed" to say "When the ICE Agent indicates that the RTCIceTransportState for an RTCIceTransport has changed _or the selected candidate pair has changed_". Otherwise changing pair while connected to another pair and still being connected would not update the pair.